pom.xml

```diff
@@ -207,6 +207,9 @@
         <galleon.log.time>true</galleon.log.time>
         <galleon.offline>true</galleon.offline>
 
+        <!-- Kubernetes / Operator -->
+        <kubernetes-client.version>5.12.0</kubernetes-client.version>
+
```

Comment:
I believe this should go to the `operator` module as well for now. This would imply that we're overriding/setting the fabric8 version in the whole project which we're not.

Reply:
done